builders/manage/sudo/impersonate.md

```diff
@@ -0,0 +1,78 @@
+
+ - A Tanssi appchain (Snap or Dedicated)
+ - Your appchain's Sudo account connected to your appchain's Polkadot.js Apps. You can refer to the [Managing Sudo guide](/builders/manage/sudo/sudo/#configuring-polkadotjs-apps){target=\_blank} for instructions on injecting your Sudo account into Polkadot.js Apps
+
+If you're unsure what your Tanssi appchain's Sudo account is, you can find it in your [Tanssi Dashboard](https://apps.tanssi.network/){target=\_blank} underneath the **Properties** section.
+
+![Locating your Sudo address on apps.tanssi.network](/images/builders/manage/sudo/impersonate/impersonate-1.webp)
+
+!!! warning
+    You should always protect your Sudo account key with the utmost security precautions, as it grants privileged access to your Tanssi appchain.
+
+## Using Sudo As {: #using-sudo-as }
+
+As you know, the Sudo account can perform privileged functions, including impersonating other accounts. When submitting a call via `sudoAs`, the runtime will first authenticate the Sudo key and then dispatch the desired function call with the `Signed` origin from a given account. In the following example, we'll use `sudoAs` to send some tokens to another account. While the result is similar to using sudo with a `forceBalanceTransfer` call, the following example uses a regular balance transfer call where the origin is the sender's account rather than the sudo account. 
+
+To make a `sudoAs` call to impersonate another account, navigate to the **Developer** Tab of Polkadot.js Apps for your Tanssi appchain and click on **Sudo**. If you do not see **Sudo** in this menu, then you have not associated the Sudo account with Polkadot.js Apps. Make sure that your Sudo account is injected by your wallet and connected to Polkadot.js Apps. Then, take the following steps:
+
+1. Select the **sudo** pallet
+2. Select the **sudoAs** method
+3. Select or paste in the desired account to impersonate
+4. Select the desired pallet for the call to submit. In this case, it is the **balances** pallet
+5. Select the **transferAllowDeath** method 
+6. Specify the destination account for the balance transfer
+7. Specify the number of tokens to send
+8. Press **SubmitSudo** and confirm the resulting pop-up
+
+![Make a sudo as call](/images/builders/manage/sudo/impersonate/impersonate-2.webp)
+
+The other account had a starting balance of `1,000` tokens prior to the call, and has subsequently dropped to `995` as expected. 
+
+![Check balances on Polkadot.js Apps](/images/builders/manage/sudo/impersonate/impersonate-3.webp)
+
+## Using Sudo and Dispatch As Utility {: #using-sudo-and-dispatch-as-utility }
+
+The following section will demonstrate using sudo to dispatch calls from an arbitrary origin. When submitting a call in this manner, the runtime will first authenticate the sudo key and then dispatch the call using the `utility` pallet and the `dispatchAs` function, allowing the transaction's origin to be exactly what you'd like.  
```

Comment:
couple cases where Sudo isn't capitalized in this para. please double-check rest of the file